src/components/chat.tsx

```diff
@@ -42,7 +42,13 @@ export function PromochatorIcon({
   imageClassName?: string;
 }) {
   return (
-    <div className="flex aspect-square justify-center rounded-full border-2 border-white">
+    <div
+      className={`flex aspect-square justify-center rounded-full border-2 border-white`}
+      style={{
+        width: `${imageWidth.toString()}px`,
+        height: `${imageHeight.toString()}px`,
+      }}
+    >
```

Comment:
**thought:** This whole approach seems messy to me but I didn't figure out how we could approach this differently yet. This works so we can leave this at that for now

Reply:
yup, I agree that it doesn't look the best